xAPI-Communication.md

```diff
@@ -295,12 +287,14 @@ results when the attachments filter is false.
     * Each additional part contains the raw data for an attachment and forms a logical part of the Statement. This 
     capability will be available when issuing PUT or POST against the Statement resource.
     * MUST include an X-Experience-API-Hash parameter in each part's header after the first (Statements) part.
-    * MUST include a Content-Transfer-Encoding parameter with a value of "binary" in each part's header after the first (statements) part.
-    * SHOULD only include one copy of an attachment's data when the same attachment is used in multiple Statements that are sent together.
-    * SHOULD include a Content-Type parameter in each part's header. For the first part (containing the statement) this MUST be "application/json".
-   	* Where parameters have a corresponding property within the Attachment Object (outlined in the table above), and both the parameter and 
-   	property are specified for a given Attachment, the value of these parameters and properties MUST match. 
-
+    * MUST include a Content-Transfer-Encoding parameter with a value of "binary" in each part's header after the first 
+     (Statements) part.
+    * SHOULD only include one copy of an attachment's data when the same attachment is used in multiple Statements that are 
+     sent together.
+    * SHOULD include a Content-Type parameter in each part's header. For the first part (containing the Statement) this 
+     MUST be "application/json".
+   	* Where parameters have a corresponding property within the Attachment Object (outlined in the table above), and both 
+   	 the parameter and property are specified for a given Attachment, the value of these parameters and properties MUST match. 
```

Comment:
we've used "attachment" and "Attachment" interchangeably in this section. I forget which is right.